tempel.el

```diff
@@ -48,9 +48,9 @@
   :group 'editing
   :prefix "tempel-")
 
-(defcustom tempel-file (expand-file-name "templates" user-emacs-directory)
-  "Path to the template file."
-  :type 'string)
+(defcustom tempel-path nil
```

Comment:
Please keep the default configuration as is `(expand-file-name "templates" user-emacs-directory)`.